doc/rst/usingchapel/chplenv.rst

```diff
@@ -700,9 +700,10 @@ CHPL_LIB_PIC
 
 Character Set
 -------------
-   We have the most experience running Chapel with the Unicode
-   character set and the traditional C collating sequence using the
-   following settings.
+   Chapel works with the Unicode character set and the traditional C collating
```

Comment:
I feel like this should mention UTF-8 somewhere in the text.

I think you should remove "Other settings might be recommended in the future" and replace it with something like "Other character sets may be supported in the future".